css/css-grid/alignment/grid-align-baseline-vertical.html

```diff
@@ -0,0 +1,121 @@
+  <div id="baseline3" data-expected-width="50" class="firstRowFourthColumn alignSelfBaseline"><img src="support/1x1-blue.png" style="width: 50px; vertical-align: middle;"></div>
+  <div data-expected-width="100" data-offset-x="0" class="firstRowFifthColumn" style="width: 100px;"></div>
+</div>
+
+<div class="grid" style="writing-mode: vertical-rl">
+  <div id="baseline4" class="firstRowFirstColumn alignSelfBaseline">ahem</div>
+  <div id="baseline5" class="firstRowSecondColumn alignSelfBaseline"><br><img src="support/1x1-blue.png" style="width: 50px;"></div>
+  <div data-expected-width="50" data-offset-x="50" class="firstRowThirdColumn alignSelfBaseline"><img src="support/1x1-blue.png" style="width: 50px; vertical-align: middle;"></div>
+  <div data-expected-width="100" data-offset-x="0" class="firstRowFourthColumn" style="width: 100px;"></div>
+</div>
+
+<div class="grid" style="writing-mode: vertical-lr">
+  <div id="baseline6" data-offset-x="13" class="firstRowFirstColumn alignSelfBaseline">ahem</div>
+  <div id="baseline7" data-offset-x="13" class="firstRowSecondColumn alignSelfBaseline"><img src="support/1x1-blue.png" style="width: 50px;"></div>
+  <div data-expected-width="50" data-offset-x="25" class="firstRowThirdColumn alignSelfCenter"><img src="support/1x1-blue.png" style="width: 50px; vertical-align: middle;"></div>
+  <div data-expected-width="50" data-offset-x="0" class="firstRowFourthColumn alignSelfBaseline"><img src="support/1x1-blue.png" style="width: 50px; vertical-align: middle;"></div>
+  <div data-expected-width="100" data-offset-x="0" class="firstRowFifthColumn" style="width: 100px;"></div>
+</div>
+
+<div class="grid" style="writing-mode: vertical-lr">
+  <div id="baseline8" class="firstRowFirstColumn alignSelfBaseline">ahem</div>
+  <div id="baseline9" class="firstRowSecondColumn alignSelfBaseline"><br><img src="support/1x1-blue.png" style="width: 50px;"></div>
+  <div data-expected-width="50" data-offset-x="0" class="firstRowThirdColumn alignSelfBaseline"><img src="support/1x1-blue.png" style="width: 50px; vertical-align: middle;"></div>
+  <div data-expected-width="100" data-offset-x="0" class="firstRowFourthColumn" style="width: 100px;"></div>
+</div>
+
+<div id="log"></div>
+
+<script>
+  var baseline1 = document.getElementById("baseline1");
+  var baseline2 = document.getElementById("baseline2");
+  var baseline3 = document.getElementById("baseline3");
+  var baseline4 = document.getElementById("baseline4");
+  var baseline5 = document.getElementById("baseline5");
+  var baseline6 = document.getElementById("baseline6");
+  var baseline7 = document.getElementById("baseline7");
+  var baseline8 = document.getElementById("baseline8");
+  var baseline9 = document.getElementById("baseline9");
+
+
+  test(() => {assert_equals(baseline1.offsetLeft, baseline2.offsetLeft, "baseline1 and baseline2 should have the same baseline;")}, "Additional Check 1");
```

Comment:
These tests may depend on some web fonts being loaded already, since the position of the items is determined based on the font family and size. 

We should call these tests as an event handler, like this:

  document.fonts.ready.then(() => {
    
      //Execute the required tests here 
  });

Another option would be to define these tests inside a function, which will be called as a handled of the body's 'onload' event, like we are doing now for the checkLayout function. Obviously, we should also move the checkLayout call to this new  function as well.